Update upgrading to slash commands tutorial (#6360)
hammerandchisel/discord-api-docs: docs/tutorials/Upgrading_to_Application_Commands.md

## Registering Commands

> Commands can be registered via HTTP requests after an app is authorized with the `applications.commands` scope. Alternatively, it's also automatically included with the `bot` scope.

Reviewer: ```suggestion
Commands can be registered via HTTP requests after an app is authorized with the `applications.commands` scope. The `applications.commands` scope is also automatically included when an app requests the `bot` scope.
```

> Commands can be registered via HTTP requests after an app is authorized with the `applications.commands` scope which can be used independently, but is also automatically included with the `bot` scope.

Reviewer 2: ```suggestion
Commands can be registered via HTTP requests after an app is authorized with the `applications.commands` scope. Alternatively, it's also automatically included with the `bot` scope.
```
